pkg/metrics/metrics.jsx

```diff
@@ -1281,36 +1280,41 @@ const PCPConfigDialog = ({
                    </>
                    }>
 
-            <Switch id="switch-pmlogger"
-                        isChecked={dialogLoggerValue}
-                        isDisabled={!s_pmlogger.exists && !packagekitExists}
-                        label={
-                            <Flex spaceItems={{ modifier: 'spaceItemsXl' }}>
-                                <FlexItem>{ _("Collect metrics") }</FlexItem>
-                                <TextContent>
-                                    <Text component={TextVariants.small}>(pmlogger.service)</Text>
-                                </TextContent>
-                            </Flex>
-                        }
-                        onChange={enable => {
-                            // pmproxy needs pmlogger, auto-disable it
-                            setDialogLoggerValue(enable);
-                            if (!enable)
-                                setDialogProxyValue(false);
-                        }} />
-
-            <Switch id="switch-pmproxy"
-                        isChecked={dialogProxyValue}
-                        label={
-                            <Flex spaceItems={{ modifier: 'spaceItemsXl' }}>
-                                <FlexItem>{ _("Export to network") }</FlexItem>
-                                <TextContent>
-                                    <Text component={TextVariants.small}>(pmproxy.service)</Text>
-                                </TextContent>
-                            </Flex>
-                        }
-                        isDisabled={ !dialogLoggerValue }
-                        onChange={enable => setDialogProxyValue(enable)} />
+            <Stack hasGutter>
+                { dialogError && <ModalError dialogError={ _("Failed to configure PCP") } dialogErrorDetail={dialogError} /> }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221201-102811-6d301c18-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)